enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationReviewBanner/ModerationReviewBanner.styled.jsx

```diff
@@ -1,22 +1,27 @@
 import styled from "@emotion/styled";
 import { color } from "metabase/lib/colors";
 import Button from "metabase/core/components/Button";
-import Icon from "metabase/components/Icon";
 
 export const Container = styled.div`
-  padding: 1rem;
-  background-color: ${props => props.backgroundColor};
+  padding: 1rem 0.5rem;
   display: flex;
   justify-content: space-between;
-  align-items: center;
-  column-gap: 0.5rem;
+  align-items: start;
+  column-gap: 1rem;
   border-radius: 8px;
 `;
 
-export const Text = styled.span`
+export const TextContainer = styled.div`
   flex: 1;
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Text = styled.span`
   font-size: 14px;
+  line-height: 17px;
   font-weight: 700;
+  margin-bottom: 4px;
```

Comment:
We tend to use `rem` for everything except `border-width`. Can this be changed to 
```
font-size: 0.875rem;
line-height: 1.0625rem;
margin-bottom: 0.25rem;
```
Could you also double check if it's possible to use `line-height: 1rem` instead?